src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/index.tsx

```diff
@@ -296,8 +372,44 @@ class DebugMeta extends React.PureComponent<Props, State> {
     );
   }
 
+  renderContent() {
+    const {searchTerm, filteredImagesByFilter: images, filterOptions} = this.state;
+
+    if (searchTerm && !images.length) {
+      const hasActiveFilter = filterOptions.find(filterOption => filterOption.isChecked);
+      return (
+        <EmptyMessage
+          icon={<IconWarning size="xl" />}
```

Comment:
Should this be IconSearch? We use that icon in other places when there are no results.